src/pages/resultsView/comparison/ComparisonTab.tsx

```diff
@@ -264,19 +264,17 @@ export default class ComparisonTab extends React.Component<
                         </MSKTab>
                     )}
                     {this.store.showGenericAssayTab &&
-                        _.keys(
+                        getSortedGenericAssayTabInfo(
                             this.store
                                 .genericAssayEnrichmentProfilesGroupedByGenericAssayType
                                 .result
-                        ).map(genericAssayType => {
+                        ).map(helper => {
                             return (
                                 <MSKTab
                                     id={`${
                                         ResultsViewComparisonSubTab.GENERIC_ASSAY_PREFIX
-                                    }_${genericAssayType.toLowerCase()}`}
-                                    linkText={deriveDisplayTextFromGenericAssayType(
-                                        genericAssayType
-                                    )}
+                                    }_${helper.genericAssayType.toLowerCase()}`}
```

Comment:
the name "helper" doesn't really help me understand what it is.

Reply:
good point, I'll try to find a better name.

Reply:
renamed as `genericAssayTabSpecs`